### ISSUE TITLE
[Screen Reader -Microbit menu controls -Advanced-Images]: Screen Reader is not reading any information for LED blocks

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2587]
User Experience 
If Screen reader is reading any information for led blocks, then the end user who rely on screen reader will find difficult to interact with the this type of controls

Test Environment
OS: RS2 Version 1703(OS Build 15063.2 
Platform: Edge.

Repro Steps 
1-Navigate to url:- https://makecode.com/
2-Navigate to the Microbit section element and select Code control given on it.
3-Navigate to the menu controls listed near the Simulator control 
4-Navigate to the Advanced control and select it
5-Navigate to the Game control and select it
6-Verify if screen reader is reading any information for led blocks or not

Actual Result
Screen Reader is not reading any information for led blocks in item mode

Expected Result
Led blocks should be descriptive

Note:
1. In Scan mode narrator is reading led blocks as 'image'.
2. NVDA is not supporting for these controls.

MAS Reference
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

Please Refer attachment file:
[MAS 4.2.1_Microbit menu controls_NotReadiungAnyThingForImages.zip](https://github.com/Microsoft/pxt/files/1187219/MAS.4.2.1_Microbit.menu.controls_NotReadiungAnyThingForImages.zip)

**Comment by veler** [comment]
This is part of Blocky, a third-part component.

**Comment by abchatra** [comment]
We have got exception for Blockly bugs from Julia Liuson (CVP). Closing these as Won't Fix. Following is the reasoning.

----------------------

MakeCode can be used in either Text editor or Block editor.

Blocks editor is inherently visual and is based on google blockly. Usage requires drag and drop functionality and we don’t have a equivalence in MakeCode for this w.r.t accessibility. 
We have evaluated couple of research projects which tries to find a equivalence to block based coding for accessibility needs: 
•	Project Torino (Requires students to buy physical blocks)
•	Accessible blockly (This is text based coding without syntax errors)
Both these are drastic departure from the blockly and provides a complete new experience. These are still in research and we intend not to use them in MakeCode at this point of time.

As workaround, the text editor in MakeCode is the way we see students with the need of accessibility using MakeCode.
